hack/test-integration.sh

```diff
@@ -43,24 +43,27 @@ runTests() {
   KUBE_GOFLAGS="-tags 'integration no-docker' " \
     KUBE_RACE="-race" \
     KUBE_TEST_API_VERSIONS="$1" \
-    KUBE_API_VERSIONS="v1" \
+    KUBE_API_VERSIONS="v1,experimental/v1alpha1" \
     "${KUBE_ROOT}/hack/test-go.sh" test/integration
 
   kube::log::status "Running integration test scenario"
 
- KUBE_API_VERSIONS="v1" "${KUBE_OUTPUT_HOSTBIN}/integration" --v=${LOG_LEVEL} --api-version="$1" \
+ # TODO: fix the integration test to make the --api-version meaningful for integration test
+ # KUBE_API_VERSIONS="v1,experimental/v1alpha1" "${KUBE_OUTPUT_HOSTBIN}/integration" --v=${LOG_LEVEL} --api-version="$1" \
+ #  --max-concurrency="${KUBE_INTEGRATION_TEST_MAX_CONCURRENCY}"
+ KUBE_API_VERSIONS="v1,experimental/v1alpha1" "${KUBE_OUTPUT_HOSTBIN}/integration" --v=${LOG_LEVEL} --api-version="v1" \
   --max-concurrency="${KUBE_INTEGRATION_TEST_MAX_CONCURRENCY}"
 
   cleanup
 }
 
-KUBE_API_VERSIONS="v1" "${KUBE_ROOT}/hack/build-go.sh" "$@" cmd/integration
+KUBE_API_VERSIONS="v1,experimental/v1alpha1" "${KUBE_ROOT}/hack/build-go.sh" "$@" cmd/integration
```

Comment:
I expect integrations test to fail atleast since we do not expose experimental by default